torch_xla/experimental/stablehlo_saved_model.py

```diff
@@ -104,10 +108,12 @@ def export_torch_model(model: torch.nn.Module,
   args = tuple(
       tree_map_only(torch.Tensor, lambda x: x.to(device=device), sample_inputs))
   orig_state_dict = copy.copy(model.state_dict())
+  orig_state_dict = tree_map_only(torch.Tensor, lambda x: x.numpy(),
```

Comment:
To align with the contract at https://github.com/pytorch/xla/blob/master/torch_xla/experimental/stablehlo_saved_model.py#L196, convert model weights to np.ndarray.